README.md

```diff
@@ -0,0 +1,42 @@
+# Template for Python-based repositories
+
+## Prerequisites
+
+- Python
+- Poetry
+- GNU Make
+- Trunk.io
+- Pyenv (optional)
+
+If you have not yet installed Trunk.io on your machine, run `make install_trunk`
+at the root of this project to install locally (works for Mac & Linux systems).
+
+## Getting started
+
+1. First things first, find all instances of `REPO_NAME` in this repo and
```

Comment:
Doing this in vscode required exluding this README.md fyi or that was updated haha